lib/stages/pulsarNetworkProcess.cpp

```diff
@@ -179,26 +179,26 @@ void pulsarNetworkProcess::main_thread() {
     clock_gettime(CLOCK_MONOTONIC, &t0);
 
     // added to take care of the missed frames
-    VDIFHeader* header = reinterpret_cast<VDIFHeader*>(packet_buffer);
-    int64_t vdif_last_seconds = header->seconds;
-    int64_t vdif_last_frame = header->data_frame;
+    PSRHeader* header = reinterpret_cast<PSRHeader*>(packet_buffer);
+    int64_t psrheader_last_seconds = header->seconds;
```

Comment:
Minor point: in some places it's `psrheader*` and in others it's `psr_header*`, for consistency I'd say just make the references here `psr_header*`

Reply:
agreed, done.